sdks/python/python_xlang_wrapper.template

```diff
@@ -0,0 +1,36 @@
+{#
+   Licensed to the Apache Software Foundation (ASF) under one
+   or more contributor license agreements.  See the NOTICE file
+   distributed with this work for additional information
+   regarding copyright ownership.  The ASF licenses this file
+   to you under the Apache License, Version 2.0 (the
+   "License"); you may not use this file except in compliance
+   with the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+#}
+
```

Comment:
Module level doc? 

In particular, we should (here or in each docstring) note that the user should import this from its proper location in the beam tree, not this module directly. (E.g. I'm thinking of how it will show up in the generated docs).

Reply:
Talked to @tvalentyn offline about this. We'll have a docstring for each module (ie. each destination) mentioning where to import these wrappers from. 

e.g. `external_transforms/io.py` will have a docstring telling the user to import these wrappers `from apache_beam.io`